bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/timeacct/TimeAcctPio.java

```diff
@@ -15,49 +16,53 @@
 
     private static final String ALLOC_PARAM_SET = "timeAcctAlloc";
     private static final String CATEGORY_PARAM  = "category";
-    private static final String HOURS_PARAM = "hours";
+    private static final String PROGRAM_PARAM   = "program";
+    private static final String PARTNER_PARAM   = "partner";
 
     private TimeAcctPio() {
     }
 
     public static ParamSet getParamSet(final PioFactory factory, final TimeAcctAllocation allocation) {
-        ParamSet pset = factory.createParamSet(TIME_ACCT_PARAM_SET);
+        final ParamSet pset = factory.createParamSet(TIME_ACCT_PARAM_SET);
 
         for (TimeAcctCategory cat : allocation.getCategories()) {
-            Double hours = allocation.getHours(cat);
-            if (hours == 0) continue;
-            
-            ParamSet ratioParamSet = factory.createParamSet(ALLOC_PARAM_SET);
-            Pio.addParam(factory, ratioParamSet, CATEGORY_PARAM, cat.name());
-            Pio.addDoubleParam(factory, ratioParamSet, HOURS_PARAM, hours);
-            pset.addParamSet(ratioParamSet);
+            final TimeAcctAward award = allocation.getAward(cat);
+            if (award.isZero()) continue;
+
+            final ParamSet allocParamSet = factory.createParamSet(ALLOC_PARAM_SET);
```

Comment:
Much clearer here. I'm not sure why this was called `ratioParamSet` in the old code, since it didn't store ratios, from what I can tell, but just the allocated hours per category.